challenges/functions.js

```diff
@@ -26,17 +66,33 @@
 // Explain in your own words why `nestedfunction()` can access the variable `internal`.
 
 // Explanation: 
+/*
+The nested function can  inherit the arguments and variables of its containing function because it contains the scope of the outer function.
 
+    The inner function can be accessed only from statements in the outer function.
 
-const external = "I'm outside the function";
+    The inner function forms a closure: the inner function can use the arguments and variables of the outer function, while the outer function cannot use the arguments and variables of the inner function.
 
-function myFunction() {
-  console.log(external);
-  const internal = "Hello! I'm inside myFunction!";
+*/
+// const external = "I'm outside the function";
+        var person = function(name) {   // The outer function defines a variable called "name"
```

Comment:
You were only expected to explain what is a closure, no need for writing new function.